src/main.cpp

```diff
@@ -82,94 +76,47 @@ void deinit()
 void setupPHYSFS()
 {
 	FileSystem& fs = FileSystem::getSingleton();
-	const std::string separator = fs.getDirSeparator();
+	
 	// Game should be playable out of the source package on all
 	// relevant platforms.
 	#if BLOBBY_ON_DESKTOP
-		std::string baseSearchPath("data" + separator);
-	#elif (defined __ANDROID__)
-		std::string baseSearchPath(SDL_AndroidGetExternalStoragePath() + separator);
-	#elif (defined __APPLE__) || (defined __SWITCH__)
-		// iOS
-		std::string baseSearchPath(PHYSFS_getBaseDir());
+		std::string baseSearchPath("data" + fs.getDirSeparator());
+	#else
+		std::string baseSearchPath(fs.getBaseDir());
 	#endif
 
+	/*
+	set write dir
+	*/
+	std::string writeDir = fs.getPrefDir();
+	fs.setWriteDir(writeDir);
+	fs.probeDir("replays");
```

Comment:
should we change the name of  `probeDir`? I don't find it very descriptive. Maybe `ensureDir`?

Reply:
I have no opinion on this ;). I think this is something we can change.